### DISCUSSION TITLE
Handcuffs used by outpost security were marked as stolen property

### DISCUSSION THREAD

**Discussion by AnalogKnight** [FakeFishGames/Barotrauma#15049] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Outpost Security will handcuff players after taking control of them. However, the handcuffs are considered stolen property, which leads to the security checking for theft later on believing that the player is stealing and attempting to attack the player again.

### Reproduction steps

1. The player was arrested after causing some trouble at the outpost.
2. Outpost security checks the player's inventory afterward and assumes the player stole the handcuffs, thus starting the attack on the player again.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Yep, they often beat you up second time right after arrest
But i don't think it's because of handcuffs because they was marked as stolen in prev versions too but i don't remember such problems

**Reply by AnalogKnight** [reply]
I think it's because bots used to not check your inventory. But now they randomly check your inventory and hostile you if they find stolen items.

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately could not find a way to get this to occur. The handcuffs don't seem to get marked as stolen (no red hand icon), and the security doesn't seem to consider them stolen when checking for stolen items. So I suspect this requires some other conditions to happen. How was the character causing trouble specifically, did they steal or attack someone? And to double-check, this happened in single player?

**Reply by AnalogKnight** [reply]
Yes, this happens in single player.
An example of this was when I used a character infected with husk to bite a security guard, who was then arrested. I then controlled another character to uncuff the arrested character, after which the cuffs went into the character's inventory and were marked as stolen Item.

**Comment by SomeRandomNoobKekeke** [comment]
@Regalis11 
Steal something publicly -> evade arrest

https://github.com/user-attachments/assets/84888d54-823d-4a74-8f98-7bd6f6f3d683

Also the problem here is that guards beat already arrested characters,
What it caused by handcuffs being marked as stolen is just an assumption,

![Screenshot_1](https://github.com/user-attachments/assets/b4e8046f-0037-4df8-9912-92231b0dc133)

Well, they seems to get triggered by those handcuffs, but they was marked as stolen [before](https://github.com/FakeFishGames/Barotrauma/discussions/13700) and guards never attacked you again after arrest
Perhaps those new conversations steps between spotting a theft and arrest makes them miss the fact that handcuffs was set by other guard and not stolen
